cba_settings.sqf

```diff
@@ -445,12 +445,12 @@ force force ace_medical_painCoefficient = 1;
 force force ace_medical_painUnconsciousChance = 0.15;
 force force ace_medical_painUnconsciousThreshold = 0.601182;
 force force ace_medical_playerDamageThreshold = 6;
-force force ace_medical_spontaneousWakeUpChance = 0.5;
-force force ace_medical_spontaneousWakeUpEpinephrineBoost = 1.5;
-force force ace_medical_statemachine_AIUnconsciousness = false;
+force force ace_medical_spontaneousWakeUpChance = 0.4;
+force force ace_medical_spontaneousWakeUpEpinephrineBoost = 1.4;
+force force ace_medical_statemachine_AIUnconsciousness = true;
```

Comment:
Please no, keep as false or else we will have server performance issues

Reply:
Will do